### DISCUSSION TITLE
Should we freeze master and wait or keep branch migrated to sound-safety updated?

### DISCUSSION THREAD

**Discussion by kengu** [fleaflet/flutter_map#873] [General]
@johnpryan / @escamoteur - we are still waiting for flutter_image to migrate to sound null-safety. PR #870 only need a change in pubspec.yaml to a stable sound null-safe version of flutter_image and we are ready to migrate flutter_map.

Since @escamoteur's contribution is now available in a branch in our repo, I can keep it up-to-date with any changes in master. I believe the correct way forward is to keep progressing master with simple PRs to prevent a prolonged blocking.

Any thoughts on this?

**Comment by johnpryan** [comment]
The other option is to merge to master and wait to ship the next release until flutter_image is published with null safety updates. I don't think I have a strong preference.

**Reply by kengu** [reply]
Agree, I think that is the best option. @escamoteur?

**Reply by escamoteur** [reply]
the advantage of merging and now into master would be that no longer new PR site created that ignore the new null safety branch.

**Reply by kengu** [reply]
OK, I'll merge it to master soon.
